src/configurations/destinations/bq/ui-config.json

```diff
@@ -132,6 +132,60 @@
           },
           "required": false,
           "footerNote": "Note: Please specify time in UTC"
+        },
+        {
+          "type": "singleSelect",
+          "immutable": true,
+          "label": "Partition Column",
+          "value": "partitionColumn",
+          "options": [
+            {
+              "name": "Ingestion Time",
+              "value": "_PARTITIONTIME"
+            },
+            {
+              "name": "Loaded At",
+              "value": "loaded_at"
+            },
+            {
+              "name": "Received At",
+              "value": "received_at"
+            }
+          ],
+          "defaultOption": {
+            "name": "Ingestion Time",
+            "value": "_PARTITIONTIME"
+          },
+          "required": false
+        },
+        {
+          "type": "singleSelect",
+          "immutable": true,
+          "label": "Partition Type",
+          "value": "partitionType",
+          "options": [
+            {
+              "name": "Hour",
+              "value": "hour"
+            },
+            {
+              "name": "Day",
+              "value": "day"
+            },
+            {
+              "name": "Month",
+              "value": "month"
+            },
+            {
+              "name": "Year",
+              "value": "year"
+            }
```

Comment:
```suggestion
```
Not sure if year is sensible choice here

Reply:
People will mostly like to choose between Hourly and Daily. Do you think we should remove the `Monthly` and `Yearly` partition type?
cc @ktgowtham @lvrach

Reply:
We decided to only provide HOUR and DAY.